### PR TITLE
fix: Logo games accessible to logged in users #502

### PR DESCRIPTION
**Pull request by psankhe28**
### What
-  Solved the issue https://github.com/openfoodfacts/hunger-games/issues/502 by following the steps mentioned there.

### CODE REVIEW

File: src/components/ResponsiveAppBar.jsx [hunk 1]
```diff
@@ -206,7 +204,7 @@ const ResponsiveAppBar = () => {
                 if (page.url) {
                   return (
                     <MenuItem
-                      key={`Mobile-${page.translationKey}`}
+                      key={page.translationKey}
```
Comment (alexfauquette) [inline src/components/ResponsiveAppBar.jsx:207]: Why do you remove those key prefixes?

Comment (psankhe28) [reply src/components/ResponsiveAppBar.jsx:207]: Extremely sorry. I guess I did it mistakenly. I will update it right now. Sorry.

Comment (alexfauquette) [reply src/components/ResponsiveAppBar.jsx]: No problem, I was just wondering if they  were a specific reason I forget to do it

**Comment by alexfauquette** [comment]
My apologies, in the issue description I forget to also add `logos/product-search` in the list of pages to which we should remove the `devOnly`.

Since they are not anymore restricted to dev mode, could you remove them in `pages/settings/index.jsx`?

**Comment by psankhe28** [comment]
> My apologies, in the issue description I forget to also add `logos/product-search` in the list of pages to which we should remove the `devOnly`.
> 
> Since they are not anymore restricted to dev mode, could you remove them in `pages/settings/index.jsx`?

Yeah I will remove it

**Comment by psankhe28** [comment]
@alexfauquette  Please have a check and let me know if there is anything that needs to be change.

**Comment by alexfauquette** [comment]
I fixed conflicts with other merged PR

**Review by alexfauquette — APPROVED** [review]

**Pull request merged**